src/components/BrandConfigProvider/styles/createStyles.ts

```diff
@@ -56,7 +56,7 @@ type ImageSuffixes = 'Image';
 
 type AnySuffix = 'Any';
 
-type StylesBuilder<T> = (theme: Theme) => T;
+type StylesBuilder<T> = (theme: Theme) => NamedStyles<T>;
```

Comment:
Sorry I didn't catch this sooner but it looks like we weren't getting the style type intellisense when writing styles with `CreateStyles`. Now we should be able to see the following:
![image](https://user-images.githubusercontent.com/2295908/236246222-1167e55a-054e-46c6-9c34-f424507d7b39.png)